core/sort/sort.model.js

```diff
@@ -7,4 +8,38 @@ export class SortModel {
 		this.mode = 'multiple';
 		this.trigger = ['reorder'];
 	}
+
+	tryToResolveSortFormat() {
```

Comment:
i think this is unnecessary to give our user such a possibility to rewrite also, sort.service is more common staff, can you change key, value functions in sort.service instead of sort.model + sort.view?
also if you change sort.by property new invalidate is called that is not good